### PR TITLE
Feature/start message manager

### PR DESCRIPTION
**Pull request by asmodee-bellanie**
Creates the beginning of the Message Manager.

### CODE REVIEW

File: src/features/message-manager/message-manager.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import _ from 'lodash'
+import { Message } from 'discord.js'
+
+import { prefixes } from "../prefix.json";
+
+export class MessageManager {
```
Comment (C0ZEN) [inline src/features/message-manager/message-manager.ts:6]: Missing service suffix?

File: src/features/prefix.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "prefixes": [
```
Comment (C0ZEN) [inline src/features/prefix.json:2]: this file seems a bit strange to me.
Why do you need a json format?

**Review by C0ZEN — CHANGES_REQUESTED** [review]
First review.

File: src/features/message-manager/message-manager.ts (hunk 1 above)
Comment (asmodee-bellanie) [reply src/features/message-manager/message-manager.ts:6]: Will fix in next PR! ^^

File: src/features/prefix.json (hunk 2 above)
Comment (asmodee-bellanie) [reply src/features/prefix.json:2]: No need.

Will change in next PR!

**Review by C0ZEN — APPROVED** [review]

**Pull request merged**